src/offlinelogstorage/dlt_offline_logstorage.c

```diff
@@ -80,14 +80,16 @@ DLT_STATIC void dlt_logstorage_filter_config_free(DltLogStorageFilterConfig *dat
         data->ecuid = NULL;
     }
 
-    if (data->log != NULL)
-        fclose(data->log);
+
```

Comment:
Could you explain a little bit why we move the code here to other lines?
Is there any reasons for closing the gzip first and then close the log?

Reply:
Yes, the order of opening is fopen gzipopen, so I think the order of closing is best gzclose, fclose